docs/docs/quick-start.md

```diff
@@ -49,6 +49,12 @@ yarn redwood --help
 ```
 For all the details, see the [CLI reference](cli-commands.md).
 
+### GitPod
+
+The fastest way to start a new Redwood project is to use GitPod ([additional documentation for working with GitPod](./how-to/using-gitpod)).
+
+![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/redwoodjs/starter)
```

Comment:
This isn't rendering too well:

<img width="631" alt="image" src="https://github.com/redwoodjs/redwood/assets/32992335/223ff06a-0eba-4996-9975-e0edc20f18b1">